pywr/recorders/_recorders.pyx

```diff
@@ -810,7 +813,10 @@ cdef class AnnualCurtailmentRatioRecorder(AbstractAnnualRecorder):
 
         for scenario_index in self.model.scenarios.combinations:
             j = scenario_index.global_id
-            self._data[i, j] = 1 - self._actual_flow[i, j] / self._max_flow[i, j]
+            if self._max_flow[i, j] <= 0.0000000000001:
```

Comment:
Use scientific notation for this, i.e. `1e-13`.

Is this consistent with other threadsholds we've used. i.e. Why not -12 or -14?